scipy/optimize/optimize.py

```diff
@@ -186,17 +186,16 @@ def rosen(x):
     
     Examples
     --------
-    >>> import numpy as np
     >>> from scipy.optimize import rosen
     
     First, we need to create 1-D array to provide to Rosenbrock function::
     
-    >>> X = np.random.randn(10)
+    >>> X = 0.1 * np.arange(10)
     
     Now, We calculate the value of Rosenbrock function:
     
     >>> y = rosen(X)
-    >>> y
+    76.56
```

Comment:
either assign to `y` and print its value or just use `>>> rosen(X)`. You can see the error message to clarify the problem.